src/js/profile/blocks/indicator.js

```diff
@@ -22,7 +22,7 @@ export class Indicator extends ContentBlock {
         const configuration = this.indicator.chartConfiguration;
 
         if (this.hasData) {
-            let c = new Chart(this, configuration, this.indicator, groups, this.container, this.title);
+            let c = new Chart(this, configuration, this.indicator, groups, this.container, this.title, this.parent.parent.parent.config.chart_attribution);
```

Comment:
@emre2038 @michaelglenister If it's possible can we remove `parent.parent.parent.` and try to fetch profile config some other way?

Reply:
I remember this was a change I made, it doesn't look very tidy. I think an option would be to mock the chart_attribution?

Reply:
It will be much better if there is a way we can call profile config here @michaelglenister 
Profile config should be accessible right through the profile pages .. right @emre2038 ?
If it doesn't then we should actually think about setting it up somewhere